doc/ag.txt

```diff
@@ -138,16 +138,36 @@ window is opened, or what size it is. Example: >
   let g:ag_qhandler="copen 20"
 <
 
+                                                           *g:ag_lmapping_func*
+A custom command used to set key mappings for the location list. Default:
+"call ag#AgSetDefaultMappings('l')". Example: >
+  let g:ag_qhandler="call AgSetMyCustomMappings()"
+<
+
+
+                                                           *g:ag_qmapping_func*
+A custom command used to set key mappings for the quickfix window. Default:
```

Comment:
I'm not against doing that, but wouldn't that require two keymaps? One for quickfix window mode and one for location list mode? The current implementation allows for a function that can take arguments and avoid duplicate code.

Again, I'm not against switching over to something like that, since ack.vim seems to do a similar thing. I'm actually interested in the logistics.